### PR TITLE
Add DevConsoleCORSFilter to Dev UI JsonRPC WebSocket

### LINKED ISSUE #29422: DevUI - config editor doesn't notify user when something goes wrong
### Describe the bug

DevUI - config editor doesn't notify user when something goes wrong

When the value gets updated there is "Configuration update successful" window in the top right corner.

But when things do not get updated, the is no notification to the end used
One needs to open console of web browser to see something like 403 Forbidden.

Users need to be informed when their config changes do not get updated.

### Expected behavior

Users are informed when their config changes through DevUI config editor do not get updated.

### Actual behavior

One needs to open console of web browser to see something like 403 Forbidden.

### How to Reproduce?

 - Use Quarkus main + #29342
 - pick an app, use 999-SNAPSHOT, configure `quarkus.http.host=0.0.0.0`
 - open DevUI on another machine, point the browser to the instance running on the first machine
 - update anything under `/q/dev/io.quarkus.quarkus-vertx-http/config`

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

Java 17

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

Quarkus main + #29342

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by phillip-kruger**
Fix #29422

### CODE REVIEW

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]
Can you add some tests for this? It would be easy for this to get lost with refactoring in the future.

**Review by cescoffier — APPROVED** [review]

File: extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/DevUIProcessor.java [hunk 1]
```diff
@@ -63,11 +63,13 @@
 import io.quarkus.maven.dependency.GACTV;
 import io.quarkus.qute.Qute;
 import io.quarkus.runtime.util.ClassPathUtils;
+import io.quarkus.vertx.http.deployment.FilterBuildItem;
```
Comment (stuartwdouglas) [inline extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/DevUIProcessor.java:66]: I think this is unused.

**Review by rsvoboda — CHANGES_REQUESTED** [review]
I receive this when running `mvn quarkus:dev` on Quarkus main 588b612421 with patch from this PR
```
__  ____  __  _____   ___  __ ____  ______
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/
2023-05-29 11:17:45,773 INFO  [io.qua.dep.dev.IsolatedDevModeMain] (main) Attempting to start live reload endpoint to recover from previous Quarkus startup failure

2023-05-29 11:17:45,772 ERROR [io.qua.run.Application] (Quarkus Main Thread) Failed to start application (with profile [dev]): java.lang.IllegalArgumentException: Path must start with /
	at io.vertx.ext.web.impl.RouteImpl.checkPath(RouteImpl.java:378)
	at io.vertx.ext.web.impl.RouteImpl.<init>(RouteImpl.java:49)
	at io.vertx.ext.web.impl.RouterImpl.route(RouterImpl.java:86)
	at io.quarkus.vertx.http.runtime.BasicRoute.apply(BasicRoute.java:62)
	at io.quarkus.vertx.http.runtime.BasicRoute.apply(BasicRoute.java:9)
	at io.quarkus.vertx.http.runtime.VertxHttpRecorder.addRoute(VertxHttpRecorder.java:976)
	at io.quarkus.deployment.steps.VertxHttpProcessor$initializeRouter938601780.deploy_0(Unknown Source)
	at io.quarkus.deployment.steps.VertxHttpProcessor$initializeRouter938601780.deploy(Unknown Source)
	at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
	at io.quarkus.runtime.Application.start(Application.java:101)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:111)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:71)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:44)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:124)
	at io.quarkus.runner.GeneratedMain.main(Unknown Source)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.quarkus.runner.bootstrap.StartupActionImpl$1.run(StartupActionImpl.java:104)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

File: extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/DevUIProcessor.java (hunk 1 above)
Comment (phillip-kruger) [reply extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/DevUIProcessor.java:66]: It's used to get the value of `FilterBuildItem.CORS` (line 141)

**Comment by phillip-kruger** [comment]
@rsvoboda please try now

**Comment by rsvoboda** [comment]
`@rsvoboda please try now` ... not sure if you solved the trouble that on the chat about the data not loaded

**Comment by phillip-kruger** [comment]
@rsvoboda I now show a better message, and (as always) the connection indicator will also be red (left bottom corner)

**Comment by phillip-kruger** [comment]
I'll add a test case a bit later.

**Review by rsvoboda — APPROVED** [review]

**Comment by phillip-kruger** [comment]
@stuartwdouglas I added the same test as from the old dev ui.

**Review by stuartwdouglas — APPROVED** [review]

**Review by Tusenka — APPROVED** [review]

**Review by Tusenka — APPROVED** [review]

**Pull request merged**